teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -68,6 +152,43 @@ Every week, there are a couple essential things that every manager should take c
+
+These specific changes should all contribute one way or another to the "Pull Request Checklist" as shown above. This checklist has to be pasted into each pull request. When reviewing, it is important to focus on both the "macro" and the "micro" issues that need to be resolved. This check list helps reviewers raise issues and review in a more efficient manner.
+
+Stemming from an effort to be more transparent and let developers be aware of how their work contributes to the organization as a whole, this list both ensures proper workflow and a more stylistically workflow. As more and more of the list is checked off, the developer is able to see their progress. Ideally, the list for a specific PR is completely checked off by the end of the week.
+
+## _Quality Assurance Procedures for Managers and Developers \[Jason\]_
+
+We want to the developers and the managers to provide give their input for each aspect of our projects. Therefore, we need procedures that allow the developers to quickly and effectively work together to complete tasks while maintaining a standard of quality.
+
+### Procedure:
+
+* [ ] Managers and Kevin conduct code reviews through GitHub
+  * [ ] Checking Issues and contributions
+    * Each time a developer makes a Pull Request, they need to provide a list of check boxes of items they addressed for their work that week.
+    * When managers review each Pull Request, they must check to ensure that all assigned issues in the milestone were completed and the checkbox contributions match the assigned issues.
+  * [ ] Every checkbox must be addressed with a comment by the reviewer
+    * A comment can acknowledge that the developer did something correctly or give constructive feedback.
+
+## _Revision by Head of Developer Relations and President \[Jeff\]_
+
+Here at Bit, we place heavy emphasis that all content and material developed by Bit is up to our standards. In addition to material being reviewed by respective managers, the material will undergo further review by the head of Develop Relations, Kevin Vvuong, as well as the President of Bit Project, Daniel Kim. We have many developer teams at Bit and this process is necessary to ensure all material is synchronous and consistent.
+
+Kevin will occasionally audit a random Pull request from a developer to a manager and ensure proper feedback is being provided. While Kevin is auditing, he will be looking to ensure the content matches Bit Project's long term goals and is consistent with the other material. 
+
+Daneil, as president, will also audit Kevin's pull requests to ensure quality is consistent through all curriculum by Bit Project. It will conducted in a similiar process to Code Reviews performed by managers. 
```

Comment:
- typo on Daniel

Reply:
I think we shouldn't say Kevin or Daniel, since we will be replaced eventually. Say President and Head of Developer Relations